admin/load.php

```diff
@@ -336,3 +336,28 @@ function perflab_get_module_data( $module_file ) {
 
 	return $module_data;
 }
+
+/**
+ * Enable all non-experimental modules on plugin activation.
+ *
+ * @since 1.0.0
+ */
+function perflab_activation_hook() {
+	// Bail if option is already set with any value.
+	if ( false !== get_option( PERFLAB_MODULES_SETTING, false ) ) {
+		return;
+	}
+
+	$modules          = perflab_get_modules();
+	$modules_settings = perflab_get_module_settings();
+
+	foreach ( $modules as $module_name => $module_data ) {
+		if ( ! $module_data['experimental'] ) {
+			$modules_settings[ $module_name ] = array( 'enabled' => true );
+		}
+	}
+
+	update_option( PERFLAB_MODULES_SETTING, $modules_settings );
+}
+
+register_activation_hook( dirname( __DIR__ ) . '/load.php', 'perflab_activation_hook' );
```

Comment:
Would be great to add a deactivation hook, removing the option when the plugin is removed.

Reply:
That may make sense, but should be a separate enhancement. This PR/issue is only to include the non-experimental modules as active by default.